third_party/move/move-prover/tests/sources/functional/enum_abort.v2_exp

```diff
@@ -0,0 +1,110 @@
+Move prover returns: exiting with verification errors
+error: abort not covered by any of the `aborts_if` clauses
+   ┌─ tests/sources/functional/enum_abort.move:17:5
+   │
+14 │           common.z // aborts
+   │           -------- abort happened here with code 0xFC6
```

Comment:
I'm confused about this 0xFC6. The code should be the one here I think? But this doesn't match what we see. We do have this treatment of splitting in category and code, is that in play here?

https://github.com/aptos-labs/aptos-core/blob/6a7b0daab5d22d2158c17676c4927ccba53e5da3/third_party/move/move-model/src/well_known.rs#L54

Reply:
I am using `STRUCT_VARIANT_MISMATCH` defined here: https://github.com/aptos-labs/aptos-core/blob/f2a427f3f4c1b883c2a9588a1c39db515637de0d/third_party/move/move-core/types/src/vm_status.rs#L854.

Reply:
`INCOMPLETE_MATCH_ABORT_CODE` is raised when during execution there is no arm matched. However, I have not been able to test it because an incomplete match can always be caught by the compiler and I have not found a transactional test for this code. Any suggestion on how to test it?